README.md

```diff
@@ -51,102 +60,103 @@ list represents a request header hint that the server is interested in
 receiving.
 
 When an `Accept-CH` opt-in is received on the top-level navigation, same-origin
-subresource requests on that page will receive the requested hints. 
+subresource requests on that page will <!-- clients still have leeway here -- is it worth being explicit about that by saying "may" or something, here? --> receive the requested hints. 
 
 #### Example
 
-If the server's response to the navigation request includes the `Accept-CH:
+If a server's response to a navigation request includes the `Accept-CH:
 foo, bar` header, same-origin subresource requests on the page will include the
 `Sec-Foo: foo-value` and `Sec-Bar: bar-value` request headers.
 
 ### `Accept-CH-Lifetime`
 
 In order to enable delivery of hints for future navigation requests to the
-origin, on top on subresource requests, the server can use `Accept-CH-Lifetime`
+same origin (not just subresource requests) servers can use the `Accept-CH-Lifetime`
 header to communicate an integer value, which represents the number of seconds,
 indicating a preference for how long this policy should be kept by the
-browser.
+client.
 
 #### Example
 
-If the server's response to the navigation request includes the `Accept-CH:
-foo, bar` and `Accept-CH-Lifetime: 3600`, same-origin requests *on that origin*
-will include the `Sec-Foo: foo-value` and `Sec-Bar: bar-value` request headers
-for the next hour, including the navigation requests of future navigations.
+If the server's response to the top-level navigation request includes the `Accept-CH:
+foo, bar` and `Accept-CH-Lifetime: 3600` headers, same-origin requests *to that origin*
+will <!-- this clause needs to specify that it's the client doing the requesting, and again... maybe make client's ultimate control about whether to send hints / no guarantees for servers explciit --> include the `Sec-Foo: foo-value` and `Sec-Bar: bar-value` request headers
```

Comment:
Don't think we need caveats *everywhere*. Also I'll probably delete that as part of the fact that ACHL is no longer a thing